website/docs/getting-started/using.md

```diff
@@ -8,7 +8,7 @@ To use the Hook, first call it, telling it how many items your menu will have:
 const { buttonProps, itemProps, isOpen } = useDropdownMenu(numberOfItems);
 ```
 
-Take the `buttonProps` object and spread it onto a button:
+Take the `buttonProps` object and spread it onto the menu button:
```

Comment:
Is there a reason why this line was changed?

Reply:
I suggested this. The reason was because we are moving away from only allowing literal button elements to any element, and I wanted to make it clear in the documentation that props should be spread onto the element that opens the menu, not particularly "a button," which I've seen referred to as the "menu button."

Reply:
Hmm, this is just the "getting started" page, so it's meant more as a quickstart for typical usage. I think I'd rather it remain "a button" just like the `README.md` file, which also contains this sentence.